packages/dds/tree/src/test/snapshots/opFormat.spec.ts

```diff
@@ -57,21 +57,21 @@ describe("SharedTree op format snapshots", () => {
 		});
 	});
 
-	it("schema change", () => {
+	it("schema change", async () => {
 		const messages = spyOnFutureMessages(containerRuntime);
-		tree.schematize({
+		const view = await tree.viewWith({
 			schema: Point,
-			initialTree: () => new Point({ x: 0, y: 0 }),
 		});
+		view.initialize(new Point({ x: 0, y: 0 }));
 
 		takeJsonSnapshot(messages);
 	});
 
-	it("field change", () => {
-		const view = tree.schematize({
+	it("field change", async () => {
```

Comment:
I updated a few tests to use `viewWith`. In general, open to doing more testing work as part of this PR.